spec/index.bs

```diff
@@ -1958,15 +1958,31 @@ interface GPUTextureUsage {
 
             **Arguments:**
             <pre class=argumentdef for="GPUDevice/createTexture(descriptor)">
-                descriptor: Description of the {{GPUTexture}} to create.
+                |descriptor|: Description of the {{GPUTexture}} to create.
             </pre>
 
             **Returns:** {{GPUTexture}}
 
-            Issue: Describe {{GPUDevice/createTexture()}} algorithm steps.
+            1. If any of the following conditions are unsatisfied, generate a validation
+                error and return an [=invalid=] {{GPUTexture}}.
+                <div class=validusage>
+                    - 1 &le; {{GPUTextureDescriptor/size}}.width
+                    - 1 &le; {{GPUTextureDescriptor/size}}.height
+                    - 1 &le; {{GPUTextureDescriptor/size}}.depth
+                    - Let |maxDim| = {{GPUTextureDescriptor/size}}.width
+                    - if {{GPUTextureDescriptor/dimension}} == {{GPUTextureDimension/'2d'}}:
+                        |maxDim| = max(|maxDim|, {{GPUTextureDescriptor/size}}.height)
```

Comment:
Maybe something like:
```suggestion
                        Set |maxDim| = max(|maxDim|, {{GPUTextureDescriptor/size}}.height)
```
mainly because right now it kind of looks like a boolean condition